src/BaseBuilder.js

```diff
@@ -122,6 +122,28 @@ export default class BaseBuilder {
   }
 
   /**
+   * Set the `errorTemplateCallback` function which will be used to
+   * render the error message for the field.
+   *
+   * @param {String => ReactNode} errorTemplateCallback
+   * @return {Builder}
+   */
+  setErrorMessageTemplate(errorTemplateCallback) {
```

Comment:
Since these are callbacks like `setLazyTemplateFactory`, I think `setErrorMessageTemplate` should have lazy in the name. What do you think? (same with the other added here)

Reply:
Spoke in person – not necessary here since they aren't evaluated in `setType` or `setOptions`.